osidb/models.py

```diff
@@ -1823,7 +1823,7 @@ class FlawReferenceType(models.TextChoices):
         max_length=50,
     )
 
-    url = models.CharField(max_length=200)
+    url = models.URLField()
```

Comment:
`URLField` is a `CharField` underneath with a `max_length=200`, maybe we should bump it to `2048` to be compliant with most web servers? I feel like we're gonna have to bump this eventually otherwise.